#launchpad bug : #524761
Index: lxdm-0.4.0/src/lxdm.c
===================================================================
--- lxdm-0.4.0.orig/src/lxdm.c	2011-07-13 14:00:34.000000000 +0200
+++ lxdm-0.4.0/src/lxdm.c	2011-07-24 18:00:58.000000000 +0200
@@ -1472,7 +1472,7 @@
 	path = g_key_file_get_string(config, "base", "path", 0);
 	if( G_UNLIKELY(path) && path[0] ) /* if PATH is specified in config file */
 		replace_env(env, "PATH=", path); /* override current $PATH with config value */
-	else /* don't use the global env, they are bad for user */
+	else if(!getenv("PATH")) /* if PATH is not set */
 		replace_env(env, "PATH=", "/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin"); /* set proper default */
 	g_free(path);
 	/* optionally override $LANG, $LC_MESSAGES, and $LANGUAGE */
